iconCompiler/config.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path')
+const currentDir = process.cwd()
+
+module.exports = {
+  dirsIconsStatic: [
+    // Don't change item's order
```

Comment:
Why's that? 
It should be a **key: value object**, rather than an array accessible by [0], [1] where order could break it all.

Reply:
agreed